org.lflang/src/org/lflang/generator/c/CReactionGenerator.java

```diff
@@ -38,7 +38,7 @@
 
 public class CReactionGenerator {
     protected static String DISABLE_REACTION_INITIALIZATION_MARKER
-        = "// **** Do not include initialization code in this reaction.";
+        = "// **** Do not include initialization code in this reaction.";  // FIXME: Such markers should not exist
```

Comment:
Let's create an issue for this. The reason behind this that you explained to me in person is _really_ worrisome.

Reply:
Done